src/saving/FileHelpers.cs

```diff
@@ -36,4 +38,24 @@ public static bool DeleteFile(string path)
             return result == Error.Ok;
         }
     }
+
+    /// <param name="filesToCheck">
+    ///   The files to check. Full path needed.
+    /// </param>
+    /// <returns>
+    ///   Returns the last modified file in the list.
+    /// </returns>
+    /// <summary>
+    ///   Gets the last modified file.
+    /// </summary>
+    public static string GetLastModifiedFile(IEnumerable<string> filesToCheck)
+    {
+        return filesToCheck.OrderBy(GetModifiedDate).FirstOrDefault();
```

Comment:
There is still the problem of making a new File object for *each* iteration, and disposing it also in each iteration.

Reply:
but I need to make it somewhere to get the date

Reply:
Create the File object before, and use it in the loop.